exportlegends.lua

```diff
@@ -1076,8 +1110,12 @@ function export_site_maps()
     end
 end
 
--- check if a folder with this name could be created or already exists
+-- Check if a folder with this name could be created or already exists
 function create_folder(folder_name)
+    if folder_name == "-00000-01-01" then
+        qerror('"'..folder_name..'" is the default foldername, this folder will not be created as you are probably not in the legends screen.')
+        return false
```

Comment:
`qerror()` raises an error (similar to `error()`), so you don't really need the `return false` here (but it doesn't hurt anything).